frontend/src/theme/buttonStyles.tsx

```diff
@@ -0,0 +1,60 @@
+import type { ComponentStyleConfig } from "@chakra-ui/theme";
+import textStyles from "./textStyles";
+
+const Button: ComponentStyleConfig = {
+  baseStyle: {
+    color: "white.default",
```

Comment:
this makes the clear all button disappear in the search and filters component 😅 
![image](https://github.com/uwblueprint/supportive-housing/assets/57375371/9b362499-2958-4c28-8ed0-4ae2a64fb759)
![image](https://github.com/uwblueprint/supportive-housing/assets/57375371/e7fd0501-9178-45a6-9596-ff992f55b685)

This doesn't have to be fixed in this PR, because the search and filters component might get removed in another PR. Just need to make note to fix this later